pyecharts/base.py

```diff
@@ -98,7 +99,7 @@ def __init__(self, title, subtitle,
             legend=[{"data": []}],
             backgroundColor=background_color
         )
-        self._jshost = constants.DEFAULT_HOST
+        self._jshost = jshost if jshost else constants.CONFIGURATION['HOST']
```

Comment:
changing _jshost would be a hack hence promote it as optional and public parameter. This enables user

1) cherry-pick one chart or a few of them to be online while others not
2) cherry-pick one chart or a few of them to use a different host while others use a common one
3) use a different host per each charts if user(super fan of this feature) would like to do so